index.js

```diff
@@ -18,7 +17,7 @@ module.exports = function(options) {
   options = options || {};
 
   var pattern = arrayify(options.pattern || ['gulp-*', 'gulp.*']);
-  var config = options.config || findup('package.json', {cwd: parentDir});
+  var config = options.config || require('findup-sync')('package.json', {cwd: parentDir});
```

Comment:
Could we still have the `var findup...` line up top? I like to have all my requires up top when possible :)